packages/formspree-core/src/core.ts

```diff
@@ -1,40 +1,37 @@
 import type { Stripe } from '@stripe/stripe-js';
-import type {
-  SubmissionData,
-  SubmissionOptions,
-  SubmissionBody,
-  SubmissionResponse,
-} from './forms';
-import {
-  appendExtraData,
-  clientHeader,
-  encode64,
-  handleLegacyErrorPayload,
-  handleSCA,
-} from './utils';
 import { Session } from './session';
+import {
+  SubmissionErrorResult,
+  SubmissionRedirectResult,
+  SubmissionStripePluginPendingResult,
+  isServerErrorResponse,
+  isServerRedirectResponse,
+  isServerStripePluginPendingResponse,
+  type FieldValues,
+  type SubmissionData,
+  type SubmissionOptions,
+  type SubmissionResult,
+} from './submission';
+import { appendExtraData, clientHeader, encode64 } from './utils';
 
 export interface Config {
+  fetch?: typeof window.fetch;
   project?: string;
   stripePromise?: Stripe;
 }
 
 export class Client {
+  private readonly fetch?: typeof window.fetch;
```

Comment:
Moved from `submitForm`. Since this is used for passing a custom implementation of `fetch`, setting it on the client, so we don't have to worry about passing the same version consistently via the option of `submitForm`.

My guess is the main use cases for this to be customizable is to mock fetch in tests (if users can't mock via something like `jest.spyOn(global, 'fetch'`) or want to pass the mock manually to the client.

Reply:
My guess is that this was primarily just for testing.

I also thought it might be for manually provide fetch (instead of using something like `isomorphic-fetch`) during server-side render. Although, to be honest, I couldn't imagine a case someone would want to submit form during SSR (maybe I miss something). That's why I wanted to double check if you have the original context/motivation for this before removing it.

Let's remove it then.